pages/index.tsx

```diff
@@ -1,75 +1,347 @@
+import { NextPage } from 'next';
 import Image from 'next/image';
+import {
+  ClockIcon,
+  SparklesIcon,
+  AdjustmentsIcon,
+  DeviceMobileIcon,
+  InboxIcon,
+  KeyIcon,
+  VideoCameraIcon,
+  BellIcon,
+} from '@heroicons/react/outline';
+import React from 'react';
 
-export default function IndexPage() {
+const features = [
+  {
+    name: 'Easy to manage requests',
+    description:
+      'Approving and declining requests is fast and straightforward. Open Overseerr, approve, and move on.',
+    icon: InboxIcon,
+  },
+  {
+    name: 'Granular Permissions',
+    description:
+      'Grant access to specific features and restrict access to others. Each user can be assigned individual permissions.',
+    icon: KeyIcon,
+  },
+  {
+    name: 'DVR Integration',
+    description:
+      'We fully integrate with the most popular DVR applications Radarr and Sonarr, including activity monitoring inside Overseerr itself!',
+    icon: VideoCameraIcon,
+  },
+  {
+    name: '4K Support',
+    description:
+      'Request standard and 4K content separately using multiple Radarr/Sonarr servers.',
+    image: (
+      <img
+        src="/icons/4k.svg"
+        className="w-6 h-6 text-white"
+        aria-hidden="true"
+      />
+    ),
+  },
+  {
+    name: 'Plex Scan',
+    description:
+      'Overseerr scans your Plex library at an interval, so we know what is already available in your libraries.',
+    image: (
+      <img
+        src="/icons/plex.svg"
+        className="w-6 h-6 text-white"
+        aria-hidden="true"
+      />
+    ),
+  },
+  {
+    name: 'Notification Agents',
+    description:
+      'Several notification agents are directly supported, such as Discord, Slack, Email, and others.',
+    icon: BellIcon,
+  },
+  {
+    name: 'Mobile Friendly',
+    description:
+      'Use Overseerr as a near-native mobile app by adding it to your homescreen. The entire app is designed to be used on any screen size.',
+    icon: DeviceMobileIcon,
+  },
+  {
+    name: 'Quotas',
+    description:
+      'Set limits to how many movies or seasons a user can request per a configurable amount of days.',
+    icon: AdjustmentsIcon,
+  },
+];
+
+interface FAQItem {
+  id: number;
+  question: React.ReactNode;
+  answer: React.ReactNode;
+}
+
+const faqs: FAQItem[] = [
+  {
+    id: 1,
+    question: 'Is there support for other languages?',
+    answer:
+      'Overseerr has already localized into over 16+ languages thanks to the community! If your language has not been localized yet, we are always accepting new translations!',
```

Comment:
```suggestion
      'Overseerr has already localized into several languages thanks to the community! If your language has not been added yet, we are always accepting new translations!',
```
Use of the word "over" means that the "+" isn't necessary. But, "over 16" still isn't quite accurate since we have exactly 16 languages on Weblate, but not all of them are fully translated.  We have 10 languages which are in the "green" and either completely or nearly completely translated, and 2 that are mostly translated.  The remaining 4 aren't really close to complete and I don't think we should factor them in to this number.

Just omitting the number and saying "several languages" avoids this issue and would allow us to not have to worry about updating this in the future. :)